src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
 			case "top-bottom (left eye first)":
 				isAnaglyph = false;
-				return Mode3D.OUL;
+				return Mode3D.ABL;
 			case "ourf":
```

Comment:
@valib Sorry, but i didn't saw that value in MediaInfo code source. They seem doesn't exist anymore.
I don't get it with the latest MediaInfo.

Are you able to get / find these values ?